community/doc_guidelines/capitalization.md

```diff
@@ -0,0 +1,153 @@
+  (Tip: Use [capitalizemytitle.com](https://capitalizemytitle.com/) and select
+  "Chicago".)
+
+    - Capitalize the first and the last word, nouns, pronouns, adjectives,
+      verbs, adverbs, and subordinate conjunctions (such as Because, Once,
+      Therefore, When, and Unless).
+    - Lowercase articles (a, an, the), coordinating conjunctions, prepositions,
+      and the "to" in infinitive verbs.
+
+* Avoid starting a sentence or title with a lowercase term, such as Splinter
+  commands (CLIs), daemons, and services (such as `splinterd`). Instead,
+  rewrite to start with "The" or some other word.
+
+* To form the plural of an all-uppercase term, add a lowercase "s" with no
+  apostrophe, such as CLIs, IDs, and REST APIs.
+
+[A](#a) - [B](#b) - [C](#c) - D - E - F - [G](#g) - H - [I](#i) - [J](#j) -
+[K](#k) - [L](#l) - [M](#m) - [N](#n) - [O](#o) - [P](#p) - Q - [R](#r) -
+[S](#s) - [T](#x) - U - V - [W](#w) - [X](#x) - [Y](#y) - [Z](#z)
+
+## A
+
+* admin service
+* API, APIs
+* application auth handler _(or application authorization handler)_
+
+## B
+
+* Biome
+
+## C
+
+* Canopy
+* CLI, CLIs
+* consensus
+
+## G
+
+* Gameroom _(example application)_
+* gameroom _(circuit in Gameroom)_
+* `gameroomd` _(daemon; always lowercase)_
+* Grid _(at first use: Hyperledger Grid)_
+* `gridd` _(daemon; always lowercase)_
+
+## I
+
+* ID, IDs
+
+## J
+
+* JavaScript
+* JSON
+
+## K
+
+* key registry
+
+## L
+
+* `libsplinter` _(always lowercase)_
+
+## M
+
+* Markdown
+* Merkle
+* Merkle-Radix tree
+
+## N
+
+* node
+* _node registry_ _(obsolete; use "registry" or "Splinter registry")_
+
+## O
+
+* OK
+* orchestrator
+
+## P
+
+* PDF _(not "pdf" or ".pdf")_
+* Pike smart contract
+* Private Counter
+* Private XO
+* Product smart contract
+
+## R
+
+* README, READMEs
+* registry _(at first use: "Splinter registry")_
+* REST API
+* Rust
+
+## S
+
+* Sabre _(at first use: Hyperledger Sawtooth Sabre)_
+* sapling
+* Sawtooth _(at first use: Hyperledger Sawtooth)_
+* scabbard service
+* `scabbard` command _(always lowercase)_
+* SCAR file - smart contract archive file _(not "scar file" or ".scar file")_
+* Schema smart contract
+* SDK, SDKs
+* Seth _(at first use: Hyperledger Sawtooth Seth)_
+* service names: admin service, scabbard service
+* smart contract
+* Splinter
+* `splinter` command _(always lowercase)_
+* `splinter` Rust crate _(always lowercase)_
+* `splinterd` _(daemon; always lowercase)_
+* state
+* state delta export
+
+## T
+
+* tic-tac-toe _(with two hyphens)_
+* TOML file _(not "toml file" or ".toml file")_
+* Transact _(at first use: Hyperledger Transact)_
+* two-phase commit protocol (or "consensus", depending on context) _(avoid "2PC")_
+
+## W
+
+* WASM _(at first use: "WebAssembly (WASM)")_
+* WebAssembly
+
+## X
+
+* XO smart contract
+* `xo` Sawtooth command _(always lowercase)_
+
+## Y
+
+* YAML file, YAML format (not "yaml file" or ".yaml file")
+
+## Z
+
+* ZeroMQ, ZMQ, 0MQ, ØMQ _(To do: Identify the preferred term)_
```

Comment:
Is the todo intentionally left in here?

Reply:
Yes, but it's probably better not to expose internal TO DO items.  Deleted.